core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemUpgradeSynchronizationRequest.java

```diff
@@ -69,6 +75,18 @@ public Boolean execute(RepositoryContext context) {
 			.setSquash(false)
 			.build()
 			.execute(context);
+		
+		if (merge.getStatus() != Merge.Status.COMPLETED) {
+			// report conflicts
+			ApiError apiError = merge.getApiError();
+			Collection<MergeConflict> conflicts = merge.getConflicts();
+			context.log().error("Failed to sync source CodeSystem content to upgrade CodeSystem. Error: {}. Conflicts: {}", apiError.getMessage(), conflicts);
+			throw new ConflictException("Upgrade code system synchronization can not be performed due to conflicting content errors.")
+				.withAdditionalInfo(Map.of(
+					"conflicts", conflicts,
+					"mergeError", apiError.getMessage()
```

Comment:
Does the `ApiError` instance from the merge request contain anything else that would be useful in eg. its `additionalInfo` property? If so, we could embed the entire error here.

Reply:
I think this is enough for now, the stack is not that useful here, message and conflicts are. Merging.